onnx/onnxifi_loader.c

```diff
@@ -128,6 +131,7 @@ int ONNXIFI_ABI onnxifi_load(
 #endif
 #endif /* ONNXIFI_LOADER_LOGGING */
 
+	printf("failed path\n");
```

Comment:
Please don't modify ONNXIFI loader either. If you want error reporting, define `ONNXIFI_LOADER_LOGGING=1` via `target_compile_definitions`

Reply:
oh sry this is only for debugging, I havent erase them yet.